src/transactions/TransactionFrameBase.h

```diff
@@ -22,9 +22,14 @@ class AbstractLedgerTxn;
 class Application;
 class Database;
 class OperationFrame;
+class TransactionFrame;
+class FeeBumpTransactionFrame;
+
+class MutableTransactionResultBase;
+using MutableTxResultPtr = std::shared_ptr<MutableTransactionResultBase>;
```

Comment:
nit: I'm not sure if 'Mutable' is necessary for the result types (unless we have an 'immutable' result somewhere).